pyproject.toml

```
@@ -0,0 +1,74 @@
+[project]
+classifiers=[
+  "Operating System :: OS Independent",
+  "Intended Audience :: Science/Research",
+  "Programming Language :: Python",
+  "Programming Language :: Python :: 3",
+  "Programming Language :: Python :: 3.9",
+  "Programming Language :: Python :: 3.10",
+  "Programming Language :: Python :: 3.11",
+  'Operating System :: Microsoft :: Windows',
+  'Operating System :: Unix',
+  'Operating System :: MacOS',
+  "Topic :: Scientific/Engineering",
+]
+
+dynamic = ["version"]
+keywords = ["unstructured grids"]
+description = "Xarray-styled package for reading and directly operating on unstructured grid datasets following UGRID conventions"
+license = {file = "LICENSE.txt"}
```

```suggestion
license = {file = "LICENSE"}
```

This one's on me, my bad, your LICENSE file doesn't have a `.txt`